service/src/main/java/com/ge/predix/acs/encryption/Encryptor.java

```diff
@@ -0,0 +1,9 @@
+package com.ge.predix.acs.encryption;
+
+public interface Encryptor {
```

Comment:
discussed with Frank. Doesnt quite work because when using a store, you dont really have a 'encrypted value'.  Probably best as 2 separate services - Encryptor and SecureStorage.  You only need one or the other   (Encryptor + regular storage  OR SecureStorage)